src/main/resources/rhsm-conduit.properties

```diff
@@ -19,6 +19,7 @@ rhsm-conduit.pinhead.requestBatchSize=1000
 
 # The API key configured by the inventory service.
 rhsm-conduit.inventory-service.apiKey=changeit
+rhsm-conduit.inventory-service.host-last-sync-threshold-hours=24
```

Comment:
I'd prefer to document default here by commenting out.

Also, in the config file let's use units in the value, rather than in the property name (i.e. `rhsm-conduit.inventory-service.host-last-sync-threshold=24h`) [related to earlier feedback].